input/pagecontent/RAD-Y1.md

```diff
@@ -194,12 +200,12 @@ The Sender shall specify each inline image reference using the `<IMRRef>` XML el
 | Attribute | Optionality | Description |
 |-----------|-------------|-------------|
 | type      | R | Type of multimedia content. Value shall be 'image' |
-| id        | R | Maps to component.id in the same `Observation` resource |
+| id        | R | Maps to component.id in the same Observation Resource |
```

Comment:
The `...` is there to highlight the resource rather than plain text. I don't think I did this consistently. So I will go over all the transactions one more time at the end to make this change.